code/clue.js

```diff
@@ -1,59 +1,227 @@
 // STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+//
 
-const mrGreen = {
-  firstName: 'Jacob',
+const MrGreen = {
```

Comment:
Careful with variable naming.  Conventially, variables begin with lowercase letters in Javascript and *classes* (like `Date`) begin with uppercase letters 👍